### ISSUE TITLE
Tapas: Locked Chapters Unavailable

### ISSUE THREAD

**Issue by alexporth** [tachiyomiorg/tachiyomi-extensions#8713]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.12.1

### Android version

Android 10

### Source information

Tapas 1.2.12

### Steps to reproduce

1. Open any Tapas webtoon
2. Login to Tapas through the webviewer
3. Exit the webviewer
4. Refresh the webtoon in tachiyomi

### Expected behavior

Chapters that have been unlocked through Tapas should be available for reading and download.

### Actual behavior

The chapters remain unavailable for both reading and download as though no login was ever performed. The chapters previously marked as read are also now marked as unread.

### Other details

I suspect that the cause can be found in the update from Tapas 1.2.11 to 1.2.11

**Comment by alessandrojean** [comment]
Should be fixed by #8699.

**Comment by Arraiment** [comment]
@alexporth are you still experiencing this issue after updating?

**Comment by Arraiment** [comment]
Yeah it is intended behaviour for lock icons to disappear after you login. If you still want to see locks for chapters, enable the second preference under Tapas' extension settings.
